src/SearchInput/SearchInput.js

```diff
@@ -116,6 +119,7 @@ class SearchInput extends PureComponent {
             searchBtnProps,
             popoverProps,
             validationState,
+            disableStyles,
```

Comment:
It needs to be removed here as well then CI should pass :)

Reply:
Yes, removed it.